infracost-usage-example.yml

```diff
@@ -834,6 +834,10 @@ resource_usage:
     monthly_jobs_compute_dbu_hrs: 1000 # Monthly number of Jobs Compute Databricks Units in DBU-hours.
     monthly_jobs_light_compute_dbu_hrs: 2000 # Monthly number of Jobs Light Compute Databricks Units in DBU-hours.
 
+  azurerm_federated_identity_credential.my_credential:
+    monthly_active_p1_users: 100 # Monthly number of active users above the 50000 free limit if you are using the Premium P1 license.
+    monthly_active_p2_users: 100 # Monthly number of active users above the 50000 free limit if you are using the Premium P2 license.
+
```

Comment:
I think we should either:
1. Call this `monthly_additional_active_pX_users` or similar if the free amount is across all resources of this type.
2. Subtract 50,000 from this number if the free amount is per resource.

This is what we do for other resources (e.g. `azurerm_network_watcher.monthly_diagnostic_checks`).,

Reply:
will go with 2